test/testmachinery/shoots/logging/seed_logging_stack.go

```diff
@@ -397,9 +401,28 @@ func prepareShootLokiStatefulSet(shootLokiSts, gardenLokiSts *appsv1.StatefulSet
 	shootLokiSts.Spec.Template.Labels = newLabels
 	// Move the shoot Loki in the garden namespace
 	shootLokiSts.Namespace = v1beta1constants.GardenNamespace
+
+	capLokiMemoryAndCPURequests(shootLokiSts)
+
 	return shootLokiSts
 }
 
+func capLokiMemoryAndCPURequests(shootLokiSts *appsv1.StatefulSet) {
```

Comment:
The func is called `capLokiMemoryAndCPURequests` but it is not only reseting the requests, but also setting a memory limit. Probably a better would be `capLokiContainerResources`/`resetLokiContainerResources`. WDYT?